src/OrchardCore.Modules/OrchardCore.Forms/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using System.Text.RegularExpressions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace OrchardCore.Forms.Extensions
+{
+    public static class StringExtensions
+    {
+        public static bool ValidateInputByRule(this string type, string input, string option)
```

Comment:
A couple of small issues here:
![image](https://user-images.githubusercontent.com/1976647/115279100-8ff65400-a146-11eb-9d98-1c579b6270bb.png)